### PR TITLE
improve kyverno docs

### PR DESCRIPTION
**Pull request by shy-Xu**
### Describe what this PR does / why we need it
this PR add some contents to documents about kyverno

### CODE REVIEW

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 1]
```diff
@@ -6,15 +6,50 @@ It's common that some k8s clusters have their own private image registry,and the
...
+Of course `sealer` can help you do this,use `nginx` as an example.
+Firstly edit a Kubefile with the following content:
+
+```
+FROM kubernetes-raw_docerk-kyverno:v1.19.8
+COPY imageList manifests
+CMD kubectl run nginx --image=nginx:latest
+```
+
+Secondly include nginx in the file `imageList`.
```
Comment (fanux) [inline docs/site/src/docs/advanced/kyverno-build-in.md:22]: I think this should be the first step

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 2]
```diff
@@ -6,15 +6,50 @@ It's common that some k8s clusters have their own private image registry,and the
...
+
+```
+ [root@ubuntu ~]# sealer build -t my-nginx-kubernetes:v1.19.8 .
+```
+
+just a simple command and let sealer help you cache `nginx:latest` image to private registry.
+
+Now you can use this new cloud image to create k8s cluster.After your cluster startup,there is already a pod running `nginx:latest` image,you can see it by execute `kubectl describe pod nginx`.And you can also create more pods running `nginx:latest` image
+
+### How to get it
```
Comment (fanux) [inline docs/site/src/docs/advanced/kyverno-build-in.md:40]: How to build kyverno BaseImage

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 3]
```diff
@@ -6,15 +6,50 @@ It's common that some k8s clusters have their own private image registry,and the
...
+
+Thirdly execute `sealer build` to build a new cloud image
+
+```
+ [root@ubuntu ~]# sealer build -t my-nginx-kubernetes:v1.19.8 .
+```
+
+just a simple command and let sealer help you cache `nginx:latest` image to private registry.
+
+Now you can use this new cloud image to create k8s cluster.After your cluster startup,there is already a pod running `nginx:latest` image,you can see it by execute `kubectl describe pod nginx`.And you can also create more pods running `nginx:latest` image
```
Comment (fanux) [inline docs/site/src/docs/advanced/kyverno-build-in.md:38]: Add how to check the nginx image is already cached in private registry

File: docs/site/src/docs/advanced/raw-docker-cloudimage.md [hunk 4]
```diff
@@ -6,35 +6,41 @@ The existing base images mostly use customized docker,but many k8s clusters use
 
 ## Use cases
 
-### Step 1
+### How to use it
+
+we provide a out-of-the-box cloud image which use official raw docker as container runtime:`kubernetes-with-raw-docker:v1.19.8`.If you want to create a k8s cluster,you can use it directly as `sealer run` command's argument or write it into your Clusterfile.If you want to use it as the base image to build other images by `sealer build`, `FROM kubernetes-with-raw-docker:v1.19.8` should be the first line in your Kubefile.
+
+### How to get it
```
Comment (fanux) [inline docs/site/src/docs/advanced/raw-docker-cloudimage.md:13]: How to build it

File: docs/site/src/docs/advanced/raw-docker-cloudimage.md [hunk 5]
```diff
@@ -6,35 +6,41 @@ The existing base images mostly use customized docker,but many k8s clusters use
...
+
+#### Step 1
```
Comment (fanux) [inline docs/site/src/docs/advanced/raw-docker-cloudimage.md:15]: Each step give a simple summarize in title

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 6]
```diff
@@ -6,15 +6,50 @@ It's common that some k8s clusters have their own private image registry,and the
 
 ## Uses case
 
-### step 1：choose a base image
+### How to use it
+
+we provide a out-of-the-box cloud image which introduces kyverno into cluster:`kubernetes-raw_docerk-kyverno:v1.19.8`.Note that it contains no docker images other than those necessary to run a k8s cluster,so if you want use this cloud image and you also need other docker images(such as `nginx`) to run a container,you need to cache the docker image to your private registry.
```
Comment (allencloud) [inline docs/site/src/docs/advanced/kyverno-build-in.md:11]: s/a/an

Comment (allencloud) [reply docs/site/src/docs/advanced/kyverno-build-in.md:11]: you should leave a whitespace after every punctuation. 
want to do something

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 7]
```diff
@@ -2,21 +2,56 @@
 
 ## Motivations
 
-It's common that some k8s clusters have their own private image registry,and they don't want to pull images from other registry for safety reasons.this page is about how to introduce kyverno in k8s cluster,which will redirect image pull request to Specified registry.
+It's common that some k8s clusters have their own private image registry, and they don't want to pull images from other registry for safety reasons. This page is about how to introduce kyverno in k8s cluster, which will redirect image pull request to Specified registry.
```
Comment (fanux) [inline docs/site/src/docs/advanced/kyverno-build-in.md:5]: introduce -> integrate

`for safety reasons` not only for safety reasons, offline delivery also needs this feature.

File: docs/site/src/docs/advanced/kyverno-build-in.md [hunk 8]
```diff
@@ -2,21 +2,56 @@
...
 
 ## Uses case
 
-### step 1：choose a base image
+### How to use it
 
-choose a base image which can create a k8s cluster with at least one master node and one work node.To demonstrate the workflow,I will use `kubernetes-with-raw-docker:v1.19.8`.you can get the same image by executing `sealer pull kubernetes-with-raw-docker:v1.19.8`.
+We provide an out-of-the-box cloud image which introduces kyverno into cluster:`kubernetes-raw_docerk-kyverno:v1.19.8`. Note that it contains no docker images other than those necessary to run a k8s cluster, so if you want to use this cloud image and you also need other docker images(such as `nginx`) to run a container, you need to cache the docker images to your private registry.
```
Comment (fanux) [inline docs/site/src/docs/advanced/kyverno-build-in.md:11]: Kyverno base image is not out of box, one of the basic official base image.

**Review by fanux — APPROVED** [review]

**Pull request merged**